src/main/java/seedu/billboard/commons/core/date/DateInterval.java

```diff
@@ -46,6 +49,18 @@
         return Optional.ofNullable(nameToDateIntervalMap.get(name));
     }
 
+    /**
+     * Checks that {@code String interval} is a valid {@code DateInterval}
+     */
+    public static boolean isValidDateInterval(String interval) {
```

Comment:
Doesn't the function `intervalFromName` achieve the functionality of this method + the static method defined in ParserUtil? Also Enum.valueOf() only accepts the exact string representation of the enum, aka "MONTH", "DAY" and so on. Is that intended behavior, to get the user to input the intervals in capital letters?

Reply:
The user is able to input intervals in both lower and upper case as the parseInterval() method converts the input to uppercase before calling Interval.valueOf()